INSTALLING.rst

```diff
@@ -1,20 +1,28 @@
 Installation
 ============
 
-**gsd** binaries are available in the `glotzerlab-software <https://glotzerlab-software.readthedocs.io>`_
-`Docker <https://hub.docker.com/>`_/`Singularity <https://www.sylabs.io/>`_ images and in packages on
-`conda-forge <https://conda-forge.org/>`_ and `PyPI <https://pypi.org/>`_. You can also compile **gsd** from source,
-embed ``gsd.c`` in your code, or read gsd files with a pure Python reader ``pygsd.py``.
+**gsd** binaries are available in the glotzerlab-software_ Docker_/Singularity_ images and in
+packages on conda-forge_ and PyPI_. You can also compile **gsd** from source, embed ``gsd.c`` in
+your code, or read gsd files with a pure Python reader ``pygsd.py``.
+
+.. _glotzerlab-software: https://glotzerlab-software.readthedocs.io
+.. _Docker: https://hub.docker.com/
+.. _Singularity: https://www.sylabs.io/
+.. _conda-forge: https://conda-forge.org/
+.. _PyPI: https://pypi.org/
 
 Binaries
 --------
 
-Anaconda package
-^^^^^^^^^^^^^^^^
+Conda package
+^^^^^^^^^^^^^
+
+**gsd** is available on conda-forge_ on the *linux-64*, *linux-aarch64*, *linux-ppc64le*, *osx-64*,
+and *win-64* platforms. To install, download and install miniforge_ or miniconda_ Then install
```

Comment:
@joaander Do you want me to try and make a conda build for Mac M1? It’s possible on conda-forge.

Reply:
Yes, that would be a great!